frontends/p4/moveDeclarations.cpp

```diff
@@ -180,6 +180,8 @@ const IR::Node *MoveInitializers::postorder(IR::ParserState *state) {
 }
 
 const IR::Node *MoveInitializers::postorder(IR::Path *path) {
+    if (!findContext<IR::ParserState>()) return path;
```

Comment:
Should check `parsersOnly` here and only return if it is false?  Otherwise it will disable this pass in controls

Reply:
The purpose of this `postorder` function is to replace references to the `start` state in parsers. We don't need that to happen in controls.

Reply:
Also, `parsersOnly` belongs to `MoveDeclarations`, not `MoveInitializers`.

Reply:
Yes, it seems like this code triggering in controls is likely what causes the sanitizer issues.  So this is likely the right fix.